SA/design_concept/content/poc/ui.typ

```diff
@@ -1,2 +1,94 @@
 = UI
-Content: interesting things about the UI (why no FRP)
\ No newline at end of file
+This chapter describes the features included in the PoC application UI, the high-level implementation and how functional reactive programming could be applied to VisualFP.
+
+== Features
+The UI for the PoC application includes two main components, which can be seen in @ui-empty-editor: A sidebar with pre-defined value blocks and the function editor.
+
+#figure(
+  image("../../static/ui_empty_editor.png"),
+  caption: "Undefined function value in the VisualFP UI"
+) <ui-empty-editor>
+
+In the PoC the user only has the possibility define one function "userDefinedFunction", which starts with a generic type hole.
+Starting with a generic function type allows more flexible testing.
+In a full version of the application, the user would be able to define the function name and type, when creating it.
+
+#grid(
+  columns: 2,
+  gutter: 5pt,
+  [#figure(
+    image("../../static/ui_editor_drag_lambda.png"),
```

Comment:
There are weird corners in the screenshots:

![image](https://github.com/VisualFP/docs/assets/20628550/7990eaab-49e4-47b9-9824-8bb4c8490ed4)

Can you see them as well?

Reply:
I think all three screenshots were affected. It is now resolved for this particular one.